src/modules/contract/contract.service.ts

```
@@ -64,54 +78,80 @@ export class ContractService {
+    }
+    const maxSize = 10 * 1024 * 1024; // 10MB
+    if (file.size > maxSize) {
+      throw new BadRequestException('File too large');
+    }
+
     const contract = this.contractRepository.create({
       title: file.originalname,
       filename: file.filename,
       contractType,
-      status: 'DRAFT'
+      status: 'DRAFT',
```

Biome (1.9.4)

[error] This property value named status is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named status


(lint/suspicious/noDuplicateObjectKeys)